src/test/scala/uk/gov/hmrc/test/ui/cucumber/PageObject.scala

```diff
@@ -19,14 +19,15 @@ import com.typesafe.config.{Config, ConfigFactory}
 import uk.gov.hmrc.test.ui.conf.TestConfiguration
 
 trait PageObject {
-  val config: Config          = ConfigFactory.load()
-  val env: String             = config.getString("environment")
-  val defaultConfig: Config   = config.getConfig("local")
-  val envConfig: Config       = config.getConfig(env).withFallback(defaultConfig)
-  val rootUrl: String         = TestConfiguration.url("pillar2-frontend")
-  val p2SubRootUrl: String    = TestConfiguration.url("pillar2-submission-frontend")
-  val back                    = "backLink"
-  val submit                  = "submit"
+  val config: Config = ConfigFactory.load()
+  val env: String = config.getString("environment")
+  val defaultConfig: Config = config.getConfig("local")
+  val envConfig: Config = config.getConfig(env).withFallback(defaultConfig)
+  val rootUrl: String = TestConfiguration.url("pillar2-frontend")
+  val p2SubRootUrl: String =
+    TestConfiguration.url("pillar2-submission-frontend")
+  val back = "backLink"
+  val submit = "submit"
   val url: String
 
```

Comment:
Formatting has changed the reading easiness. Can we use the formatting which is easy to read?